### PR TITLE
[1LP][RFR] fix cfme/tests/infrastructure/test_instance_analysis.py

### PR DESCRIPTION
**Pull request by otsuman**
{{ pytest: cfme/tests/infrastructure/test_instance_analysis.py --long-running}}

1 test failed with assertion error because after SSA it shows 0 files, possible bug and not automation failure

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 971be5908180247e88af5c3d8a98cd02adb50761:

`cfme/configure/configuration/__init__.py`:
- :red_circle: Line 237:47: **F841** *local variable 'CandidateNotFound' is assigned to but never used*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/common/vm.py [hunk 1]
```diff
@@ -717,29 +718,15 @@ def equal_drift_results(self, row_text, section, *indexes):
...
-        for other_section in sec_loc_map.keys():
-            other_section_loc = sec_loc_template.format(sec_loc_map[other_section])
-            other_section_classes = sel.get_attribute(other_section_loc + '/..', "class")
-            if other_section != section and 'dynatree-partsel' in other_section_classes:
-                # Element needs to be checked out if it has no dynatree-selected
-                if 'dynatree-selected' not in other_section_classes:
-                    sel.click(other_section_loc)
-                sel.click(other_section_loc)
+        for other_section in drift_section.child_items():
+            drift_section.check_node(other_section.text)
```
Comment (psav) [inline cfme/common/vm.py]: Check then uncheck the same node? Perhaps needs some explanation...

File: cfme/web_ui/__init__.py [hunk 2]
```diff
@@ -3440,9 +3440,11 @@ def cell_indicates_change(self, row_text, col_index):
 
         # Cell either contains an image
         try:
-            cell_img = sel.element(".//img", root=cell)
-            if sel.get_attribute(cell_img, "alt") == 'Changed from previous':
-                return True
+            if version.current_version() >= '5.8':
```
Comment (psav) [inline cfme/web_ui/__init__.py]: To avoid a version pick, can't we just make the expression an OR for the xpath?

File: cfme/web_ui/__init__.py [hunk 3]
```diff
@@ -3418,7 +3418,7 @@ def get_cell(self, row_text, col_index):
             Selenium element of the cell.
         """
         self.expand_all_sections()
-        cell_loc = ".//div/div[1][contains(., '{}')]/../div[{}]".format(row_text, col_index + 2)
+        cell_loc = ".//th[contains(., '{}')]/../td[{}]".format(row_text, col_index + 1)
```
Comment (izapolsk) [inline cfme/web_ui/__init__.py]: REQUIRED: please don't forget to use normalize-space in cases like that.

File: cfme/common/vm.py (hunk 1 above)
Comment (otsuman) [inline cfme/common/vm.py]: It is hierarchic tree and some child items selected but not all of them, so it is done to  check all of them

**Comment by izapolsk** [comment]
@otsuman, LGTM but one test fails in PRT. Could you explain this ?

**Comment by otsuman** [comment]
@izapolsk in PR description. 
1 test failed with assertion error because after SSA it shows 0 files, possible bug and not automation failure

**Pull request merged**